src/main/java/com/imsweb/datagenerator/naaccr/rule/tumor/PhysicianRule.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018 Information Management Services, Inc.
+ */
+package com.imsweb.datagenerator.naaccr.rule.tumor;
+
+import java.util.List;
+import java.util.Map;
+
+import com.imsweb.datagenerator.naaccr.NaaccrDataGeneratorOptions;
+import com.imsweb.datagenerator.naaccr.NaaccrDataGeneratorRule;
+import com.imsweb.datagenerator.utils.RandomUtils;
+
+public class PhysicianRule extends NaaccrDataGeneratorRule {
+
+    // unique identifier for this rule
+    public static final String ID = "physician";
+
+    /**
+     * Constructor.
+     */
+    public PhysicianRule() {
+        super(ID, "Physician fields");
+    }
+
+    @Override
+    public void execute(Map<String, String> record, List<Map<String, String>> otherRecords, NaaccrDataGeneratorOptions options, Map<String, Object> context) {
+        if ((options != null) && (options.getPhysicians() != null)) {
```

Comment:
You have extra parenthesis here (the inner ones are not needed).

And in addition to the list being null, I think you should also check it's not empty:
```java
if (options != null && options.getPhysicians() != null && !options.getPhysicians().isEmpty())
```
I assume the code the is the same for the facility rules and so this applies there as well.